sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/planning/patterns.scala

```diff
@@ -391,35 +393,41 @@ object PhysicalWindow {
   }
 }
 
-object ExtractSingleColumnNullAwareAntiJoin extends JoinSelectionHelper with PredicateHelper {
-
-  // TODO support multi column NULL-aware anti join in future.
-  // See. http://www.vldb.org/pvldb/vol2/vldb09-423.pdf Section 6
```

Comment:
How about keeping this comment for the reference to NAAJ?  I think this is a good material to understand how it works. (NOTE: IMHO the link to [the ACM page](https://dl.acm.org/doi/10.14778/1687553.1687563) is better than the direct link to the PDF).